demo/src/main/java/com/example/demo/models/User.java

```diff
@@ -0,0 +1,31 @@
+package com.example.demo.models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Getter
+@Setter
```

Comment:
Use lombok annotations `@Setter` and `@Getter` to avoid generating boiler plate code (like getters and setters), because it's just a noise that reduce code readability